src/BenchmarkDotNet.Core/Jobs/JobExtensions.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using BenchmarkDotNet.Toolchains;
+using BenchmarkDotNet.Characteristics;
+using BenchmarkDotNet.Engines;
+using BenchmarkDotNet.Environments;
+using BenchmarkDotNet.Horology;
+
+namespace BenchmarkDotNet.Jobs
+{
+    public static class JobExtensions
```

Comment:
It's ok. When we are working with Jobs, we usually don't care about allocations. =)